Sources/FuzzilliCli/main.swift

```diff
@@ -297,7 +299,7 @@ fuzzer.sync {
             let newProgs = try decodeProtobufCorpus(data)
             logger.info("Starting Cov-only corpus input of size \(newProgs.count). This may take some time")
             fuzzer.importCorpus(newProgs, importMode: .newCoverageOnly)
-            logger.info("Successfully imported coverage only corpus import. Corpus now contains \(fuzzer.corpus.size) elements")
+            logger.info("Successfully imported coverage only corpus import. Samples will appear in corpus as they are asynchronously minimized")
```

Comment:
Nit: maybe remove "import" after "coverage only corpus"? Also, maybe make it "Samples will appear in the corpus once they have been minimized", it's probably not necessary to restate that this happens asynchronously here :)